cron/update_questions.py

```diff
@@ -1,16 +1,12 @@
 import os
 import json
-import requests
 from datetime import datetime
+import leetcode
+import leetcode.auth
 
-query = '''query questionData($titleSlug: String!) {
-  question(titleSlug: $titleSlug) {
-    difficulty
-  }
-}
-'''
+LEETCODE_SESSION_TOKEN = os.environ.get("LEETCODE_SESSION_TOKEN")
 
-questions_file = os.getcwd() + "/src/data/questions.json"
+questions_file = "../src/data/questions.json"
```

Comment:
We'll need to leave this in - it's needed for when our GitHub Action clones the repo into the container that is spun up. The current working directory will be the root of our repo (ie. `~/leetcode-patterns/`, not the root of this file (ie. `leetcode-patterns/cron/update_questions.py`.

Reply:
Yep, I just fixed it!